### ISSUE TITLE
How to run the tests

### ISSUE THREAD

**Issue by gs-spadmanabhan** [rafaelfirmino/amqp-support#1]
Just stumbled upon this git repo and I am unable to figure out how to run the test case. Could help me out?

**Comment by rafaelfirmino** [comment]
Hi. 

The examples in the test/java/br/com/pexin/amqp samples package are just to show how to use the annotation.

To run the project you must import the amqp-support project as a dependency of your main project, in it you will create a producer or consumer class like the samples, the same annotations.

In your_project -> import -> amqp-support

In your_project -> (spring-amqp dependencies and spring-amqp settings with your connections, ip's and port to your Rabbit instance and etc)

In your_project -> classes using the producer and consumer annotations created in amqp-support

**Comment by gs-spadmanabhan** [comment]
Thank you for the response,

Here is what I did, In this app itself I created an App class and annotated it with @SpringBootApplication. I copied the TestConsumer and Producer.  Then I initialized consumer.start() and producer.sendMessage() but I am getting some autowire issue.

I am pretty new the spring amqp hence an example with Spring Boot Application sample will help me to understand the flow (through debugging).
